students/Sean_Tasaki/Lesson2/series.py

```diff
@@ -0,0 +1,83 @@
+def fibonacci(n):
+    """Return the nth value in the fibonacci sequence (starting with zero index.
+    """
+    if n == 1:
+        return 1
+    if n == 0:
+        return 0
+    if n < 0:
+        print("Invalid data")
+    else:
+    	return fibonacci(n - 2) + fibonacci(n - 1)
+
+def lucas(n):
+	"""Return the nth value in the Lucas numbers series (starting with zero index.
+	"""
+	if n == 1:
+		return 1
+	if n == 0:
+		return 2
+	if n < 0:
+		print("Invalid data")     
+	else:
+		return lucas(n - 2) + lucas(n - 1)
+
+def sum_series(n, index0 = 0, index1 = 1):
+	"""Return the nth value in a series. The default parameter values set to calculate
+	fobonacci series. Calling with optional arguments index0 = 2 and index1 = 1
+	will calculate lucas series.
+	"""
+	# Calculates fibonacci series
+	if (index0 == 0 and index1 == 1):
+		if n == 1:
+			return 1
+		if n == 0:
+			return 0
+			if n < 0:
+				print("Invalid data")
+		else:
+			return fibonacci(n - 2) + fibonacci(n - 1)
+    # Calculates lucas series
+	elif (index0 == 2 and index1 == 1):
+		if n == 1:
+			return 1
+		if n == 0:
+			return 2
+		if n < 0:
+			print("Invalid data")     
+		else:
+			return lucas(n - 2) + lucas(n - 1)	
```

Comment:
The goal for the `sum_series` function is to default to fibonacci, but to be able to generate a new sequence based on any two input parameters (not just the lucas series). You can do this by using your variable parameter values (`index0` and `index 1`) in place of the hard coded values that you used for the first two items of the other sequences.